requirements/requirements-cv.txt

```diff
@@ -1,6 +1,6 @@
 albumentations>=0.4.3
 imageio
-kornia>=0.3.1
+git+https://github.com/kornia/kornia.git@b9ffe7efcba7399daeeb8028f10c22941b55d32d#egg=kornia
```

Comment:
todo: fix this later

Reply:
@Scitator we are releasing later this week `v0.4.1`. What's the actual issue here ? /c @shijianjian